Documents/flashing

```diff
@@ -6,17 +6,21 @@
 
 Visually identify any content that flashes. Flashing is content that rapidly alternates between two or more states that vary significantly in contrast.
 
-When flashing content IS found, the result for Test ID 3.A <span id="OLE_LINK17" class="anchor"></span>(Test Name: 2.3.1-flashing) is **NOT TESTED**.
+-	  It is recommended that testers inquire about occurrences of flashing content. Vendors/developers should make testers aware of the location of any flashing content. (does this belong in this doc? Or e.g. guidance including sampling guide)
+-	  The modified “small safe area” is intended for normal viewing distances or equivalent. If the content is intended for large displays where users may easily decrease the viewing distance, flashing content should be limited to  … (KE: tell tester to change res/screen/browser zoom, size) cs: leave to tool? Using G176 approx, maybe 1/6 of screen?
+-	  Testers with known sensitivities to flashing content should avoid testing flashing content.
```

Comment:
what test result should be entered if the tester does not test?

Reply:
Right now it would be "not tested" but in future, ideally something like "found but not tested" to clarify that flashing content was in fact present

Reply:
If the flashing content is bigger than the "small safe area", what is the test result? 

It isn't a full test of SC 2.3.1, so it can't be a true fail. "Further testing needed" would be true, but doesn't really say anything...

Partial testing can raise as many questions as it tries to answer.

Reply:
It'd be like now, "not tested".